### DISCUSSION TITLE
How to put as image to run on local windows agent?

### DISCUSSION THREAD

**Discussion by masfaraud** [woodpecker-ci/woodpecker#4139] [Ask a Question] [answered]
I am running a local agent on windows.
I struggle with the "image" section. I have read the following:
- https://woodpecker-ci.org/docs/administration/backends/local#options
- https://github.com/woodpecker-ci/woodpecker/discussions/3467

In my case image: powershell or image: cmd does not work.
What to used?
I get:
```
Error response from daemon: pull access denied for cmd, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

**Comment by qwerty287** [comment, marked as answer]
This looks like it still uses the docker backend. Please check your `WOODPECKER_BACKEND` configuration option again
